library/core/src/intrinsics/mod.rs

```diff
@@ -2569,9 +2569,10 @@ pub const unsafe fn const_make_global(ptr: *mut u8) -> *const u8 {
 /// of not prematurely committing at compile-time to whether contract
 /// checking is turned on, so that we can specify contracts in libstd
 /// and let an end user opt into turning them on.
-#[rustc_const_unstable(feature = "contracts_internals", issue = "128044" /* compiler-team#759 */)]
 #[unstable(feature = "contracts_internals", issue = "128044" /* compiler-team#759 */)]
+#[rustc_const_unstable(feature = "contracts", issue = "128044")]
 #[inline(always)]
+#[lang = "contract_checks"]
 #[rustc_intrinsic]
```

Comment:
Making the same thing both a lang item and an intrinsic doesn't make a lot of sense. Why do you propose to do this?